tests/cnf/ran/gitopsztp/tests/IBBF-e2e-test.go

```diff
@@ -0,0 +1,152 @@
+package tests
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/openshift-kni/eco-goinfra/pkg/configmap"
+	"github.com/openshift-kni/eco-goinfra/pkg/hive"
+	"github.com/openshift-kni/eco-goinfra/pkg/reportxml"
+	siteconfigv1alpha1 "github.com/openshift-kni/eco-goinfra/pkg/schemes/siteconfig/v1alpha1"
+	"github.com/openshift-kni/eco-goinfra/pkg/siteconfig"
+	"github.com/openshift-kni/eco-gotests/tests/cnf/ran/gitopsztp/internal/gitdetails"
+	"github.com/openshift-kni/eco-gotests/tests/cnf/ran/gitopsztp/internal/tsparams"
+	. "github.com/openshift-kni/eco-gotests/tests/cnf/ran/internal/raninittools"
+	"github.com/openshift-kni/eco-gotests/tests/cnf/ran/internal/ranparam"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"fmt"
+	"time"
+)
+
+var _ = Describe(
+	"Performing Image-Based Break/Fix Flow",
+	Label(tsparams.LabelIBBFe2e), func() {
+
+		var (
+			spokeNamespace = RANConfig.Spoke1Name
+		)
+
+		AfterEach(func() {
+
+			By("Cleanup test configmap")
+
+			err := configmap.NewBuilder(HubAPIClient, tsparams.TestCMName, spokeNamespace).
+				Delete()
+			Expect(err).ToNot(HaveOccurred(), "Unable to delete configmap")
```

Comment:
What do we need in order to rerun the test in the same env without re-installation? 

The test ends with a IBBF test git dir, and already updated configmap, etc, which feels like we will not be able to rerun the test without manual reverting various things. If so, we should consider adding these manual steps in AfterEach.